go.mod

```diff
@@ -161,6 +161,7 @@ require (
 	github.com/cyphar/filepath-securejoin v0.4.1 // indirect
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/dimchansky/utfbom v1.1.1 // indirect
+	github.com/docker/docker v28.2.2+incompatible // indirect
```

Comment:
Probably we should update the version to prevent being flagged for the CVE 🤔

Reply:
well, at least until BuildKit removes the dependency;

- https://github.com/moby/moby/pull/50515